block/interface/header.go

```diff
@@ -228,4 +228,8 @@ type Header interface {
 
 	// Copy returns a copy of the header.
 	Copy() Header
+
+	Slashes() []byte
```

Comment:
comment please about this interface, what's the return value?

Reply:
why no error return?

Reply:
Added comment. No error return because it is the same convention of other similar fields, like CrossLinks. Caller deals with the nil.